### DISCUSSION TITLE
Is it possible to add Hyper rate To use instead of Pulsoid

### DISCUSSION THREAD

**Discussion by THEROCKSSS** [BoiHanny/vrcosc-magicchatbox#98] [Ideas]
I would like to ask the possible feature of this being added I know a lot of people who just want to track their heartbeat for free and do not want to be paying for it that's why I'm suggesting hyper rate using my Apple watch for it I can get an accurate heartbeat rating for VR chat with VRCOSC Which also offers Pulsoid As a module for other users to use that's why I'm asking to have this possibly be a feature added into version 2 or being the next update So please consider Adding hyper rate as an additional feature or as an add-on

**Comment by ethanboisvert6-star** [comment]
Hi
